#GlobalPlanner
get_path_service: /ras_group8_global_planner/get_path

#MoveArm
move_arm_up_service: /ras_group8_arm/cartesian_up
move_arm_down_service: /ras_group8_arm/cartesian_down

#Spearker
speaker_topic: /espeak/string

#PathExecution
send_path_topic: /ras_group8_brain/send_path
path_stop_topic: /ras_group8_brain/stop_driving
path_done_topic: /ras_group8_navigation_system/path_done
robot_position_topic: /ras_group8_navigation_system/robot_position
new_map_topic:  /ras_map/new_wall
add_object_map_topic: /ras_group8_brain/add_object_map

#ObstacleDetection
vision_msg_topic: /Object_detection


arm_up:
    pickup_attempt: 8
    pickup_range: 1

obstacle:
    position_accurancy: 0.05
    detection_accurancy: 0.10
    obstacle_try: 10
    value_group1: 100
    value_group2: 1000
    value_group3: 5000
    value_group4: 10000

home:
    obstacle_x: 0.0
    obstacle_y: 0.16
    obstacle_z: -0.1
    
    position_x: 0.15
    position_y: 0.15
    orientation_z: 0.7
    orientation_w: 0.7
    home_accurancy: 0.1

init:
    first_round: false
    obstacle_file: /home/ras18/ObstacleFile.txt
    round_time: 300
    maze_size_x: 2.40
    maze_size_y: 2.40
